packages/toolkit/upgrade/modern.config.js

```diff
@@ -1,5 +1,10 @@
-const { tscLikeBuildConfig } = require('@scripts/build');
-
 module.exports = {
-  buildConfig: tscLikeBuildConfig,
+  buildConfig: {
+    autoExternal: false,
+    alias: {
+      chalk: '@modern-js/utils/chalk',
```

Comment:
So why do we need to bundle this package? It will cause duplicate code.

For example, `@modern-js/codesmith` will be installed in the user's project, but it will also be bundled with `@modern-js/upgrade`. There will be two copies of the `@modern-js/codesmith` code.

Reply:
I get you mean. `@modern-js/upgrade` publish file now is only `dist/index.js` to run with `npx @modern-js/upgrade`. We need to support two methods.

Reply:
So why does `npx @modern-js/upgrade` need to run with only `dist/index.js`?